spec/std/process_spec.cr

```diff
@@ -237,4 +237,38 @@ describe Process do
       Process.find_executable("some_very_unlikely_file_to_exist").should be_nil
     end
   end
+
+  it "has a user" do
+    user = Process.user
+    user.should be_a(User)
+  end
+
+  it "has an effective user" do
+    user = Process.effective_user
+    user.should be_a(User)
+  end
+
+  {% if flag?(:openbsd) || flag?(:freebsd) || flag?(:linux) %}
+  it "has a saved user" do
```

Comment:
We typically indent inside macros, however the formatter is incapable of formatting macros (they're too unstructured), so they have to be formatted/indented by hand.

Reply:
Fixed